docs/reference/commandline/network_rm.md

```diff
@@ -10,13 +10,13 @@ parent = "smn_cli"
 
 # network rm
 
-    Usage:  docker network rm [OPTIONS] NAME | ID
+    Usage:  docker network rm [OPTIONS] NETWORK [NETWORK...]
 
-    Deletes a network
+    Deletes one or more networks
 
       --help=false       Print usage
 
-Removes a network by name or identifier. To remove a network, you must first disconnect any containers connected to it.
+Removes one or more networks by name or identifier. To remove a network, you must first disconnect any containers connected to it.
 
 ```bash
   $ docker network rm my-network
```

Comment:
Perhaps we should add additional examples;

- one showing removing multiple networks
- and example showing removing a network by `id`